src/app/migrations/password/password.2.angular.txt

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+@Component({
```

Comment:
What is the difference between `password.1`, `password.2` and `password.3` examples here ?

Reply:
password 1 - has no visible label
password 2 - has a label
password 3 - shows how you can use `[ngModel]` -> I have to remove the `ngModel` from 1 and 2.